arunmsharma/README.md

```diff
@@ -0,0 +1,6 @@
+Learn about c++ and java. 
+Will do competitive coding regularly.
+will try to read interview experiences.
+will try to read technical posts.
+will try to participate 4-5 contests in every month.
+will try to polish my skills as a web developer
```

Comment:
Can you please follow something similar to the format used by the people whose request is merged?
[Example](https://github.com/Codeshows/100DaysOfCode/tree/master/RashakDude)

You also need to give your basic information. Please follow the above example and user markdown for formatting.